manimpango/cmanimpango.pyx

```diff
@@ -103,13 +106,12 @@ def text2svg(
         cairo_move_to(cr,START_X + offset_x,START_Y + line_spacing * setting.line_num)
 
         pango_cairo_update_layout(cr,layout)
+        markup = escape(text_str)
+        if color:
+            markup = (f"<span color='{color}'>{markup}</span>")
```

Comment:
If `color` here is invalid, this could probably error out and we aren't catching the error. I would suggest you check with [`MarkupUtils.validate`](https://manimpango.manim.community/en/latest/reference/manimpango.MarkupUtils.html#manimpango.MarkupUtils.validate) to check whether the markup here is correct.

Reply:
I tried passing an invalid colour here. In that case, the SVG file is simply empty and Pango doesn't bail out. I guess it's better to check it beforehand.

Reply:
I've updated the code to validate color markup, it should now raise `ValueError` on invalid color.